src/web/js/trove/image.js

```diff
@@ -242,6 +242,18 @@
     var annAngle = ann("Angle (a number 'x' where 0 <= x < 360)", image.isAngle);
     var checkAngle = p(image.isAngle, "Angle");
 
+    var canonicalizeAngle = function(angle) {
```

Comment:
Not quite, that's wrong for negative angles (you'll be too small by 360 degrees).  But it's then an easy matter to fix up...